gui/resources.xml

```diff
@@ -467,21 +478,23 @@ Vocabulary
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Merge the current layer with the layer beneath it.</property>
           <signal name="activate" handler="merge_layer_down_cb"/>
         </object>
-        <accelerator key="Delete" modifiers="GDK_CONTROL_MASK"/>
+        <accelerator key="m" modifiers="GDK_CONTROL_MASK"/>
       </child>
       <child>
         <object class="GtkAction" id="MergeVisibleLayers">
           <property name="label" translatable="yes" context="Menu→Layer (labels), Accel Editor (labels)">Merge Visible Layers</property>
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Consolidate all visible layers, and delete the originals.</property>
           <signal name="activate" handler="merge_visible_layers_cb"/>
         </object>
+        <accelerator key="Delete" modifiers="GDK_CONTROL_MASK | GDK_SHIFT_MASK"/>
       </child>
       <child>
         <object class="GtkAction" id="NewLayerMergedFromVisible">
           <property name="label" translatable="yes" context="Menu→Layer (labels), Accel Editor (labels)">New Layer from Visible</property>
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Like ‘Merge Visible Layers’, but doesn’t delete the originals.</property>
           <signal name="activate" handler="new_layer_merged_from_visible_cb"/>
         </object>
+        <accelerator key="m" modifiers="GDK_CONTROL_MASK | GDK_SHIFT_MASK"/>
```

Comment:
No not really. The reason I would have a shortcut for this is to allow me to test what the layer would look like before I commit to merging the layers.

Though now that you say that, I think having a `MergeGroup` layer option would be nice. Plus having `NewGroupFromVisible` tool. I'll have to look into that in the future.

So I'm guessing since it is an advance feature(like the Normalize Layer) it should be up to the user to manually click it in the menu?